### PR TITLE
2.1.0 Release Notes

### PR DESCRIPTION
**Pull request by lemieux**
Here ya go. 

@Mario54 @mspensieri @jpjoyal @alavers

### CODE REVIEW

File: release_notes/v2.1.0.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Money this way please!
+
+## Buy actions
+
+![money](https://media0.giphy.com/media/8LDwLDu12y6Fa/200_s.gif)
```
Comment (jpjoyal) [inline release_notes/v2.1.0.md]: this is not animated

File: release_notes/v2.1.0.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+You can now request money from your users. Payments are handled through [Stripe](https://stripe.com/), and setup is extremely simple. Read all the details [in our docs](http://docs.smooch.io/javascript/#stripe).
+
+
+## Changes
+- The widget was fully rewritten using React and Redux. The public API is the same, but the events published slightly changed. The widget was previously publishing `message:received` and `message:sent`. Now, it only publishes `message` whenever a new message is added to the conversation. You can always check who sent that message by checking the `role` attribute for the message.
```
Comment (jpjoyal) [inline release_notes/v2.1.0.md]: Don't forgot to update this once you add back the `message:received` and `message:sent` events

File: release_notes/v2.1.0.md (hunk 1 above)
Comment (lemieux) [inline release_notes/v2.1.0.md]: blame the guy who uploaded it

File: release_notes/v2.1.0.md (hunk 2 above)
Comment (lemieux) [inline release_notes/v2.1.0.md]: Done, I also brought back the `destroy` event

File: release_notes/v2.1.0.md (hunk 1 above)
Comment (mspensieri) [inline release_notes/v2.1.0.md]: http://www.reactiongifs.com/r/mone.gif

**Comment by mspensieri** [comment]
:+1:

File: release_notes/v2.1.0.md (hunk 1 above)
Comment (lemieux) [inline release_notes/v2.1.0.md]: fixed

**Comment by jpjoyal** [comment]
:+1:

**Comment by alavers** [comment]
belated :+1:

**Pull request merged**